utils/version/version.inc

```diff
@@ -0,0 +1,52 @@
+#pragma once
```

Comment:
Should we be checking in this file? It will be generated by the build, won't it?

Reply:
There is a -fv option on hctbuild that enables overiding of the generated version and this is the default file it reads the version info from. The location can be overriden by the new -fvloc option but -fv needs to work on its own (I believe AppVeyor uses it and that's how we versioned the releases until now). It's a new file because the fixed version was previously specified as strings in AddLLVM.cmake. Now we don't need to touch cmake files to change the version.

Reply:
Ok! So the file generated by the build has the same format but will be somewhere else?

Reply:
Same format and same location (<build_dir>\utils\version), but it is either going to be generated or copied from here.